README.md

```diff
@@ -0,0 +1,28 @@
+# Pterodactyl API Documentation
+This repository includes the documentation for the [Pterodactyl panel](https://pterodactyl.io) API.
+The documentation, files of which are in this repository,
+is hosted at [api.ptero.sh](https://api.ptero.sh/#).
+
+# Structure
+This repository is structures in two main parts:
+## OpenAPI spec
+The OpenAPI specifications (the Client API [/api/client] and the Application [/api/application] specifications)
+are contained in the [`reference`](./reference) directory.
+## Articles
+The articles are an extension of the OpenAPI spec,
+and are additional documentation for, for example, the WebSocket API.
+Currently there is one article:
+- [WebSocket API](./docs/Server-Websockets.md)
+
+# Libraries
+This list includes libraries that help developers interact with the Pterodactyl API. \
+More entries will be added soon.
+
```

Comment:
This is not what was meant by including API libraries/wrappers...

Reply:
Then please tell me what you actually ment.

Reply:
Having a dedicated page in the docs for API libraries/wrappers, but there still needs to be more discussion on what repositories get added to it.

Reply:
For now, this would be a section in readme. There aren't enough good wrappers. For now it'd just be dartactyl

Reply:
we could potentially consider adding a section somewhere that mentions all known wrappers and how much of the API they cover (ex application, client, includes websocket wrapper, etc) and sort them on the page based on that